app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+from rest_framework import serializers
+
+from experimenter.experiments.models import (
+    NimbusBranch,
+    NimbusBucketRange,
+    NimbusExperiment,
+)
+
+
+class NimbusBucketRangeSerializer(serializers.ModelSerializer):
+    randomizationUnit = serializers.ReadOnlyField(
+        source="isolation_group.randomization_unit"
+    )
+    namespace = serializers.ReadOnlyField(source="isolation_group.namespace")
+    total = serializers.ReadOnlyField(source="isolation_group.total")
+
+    class Meta:
+        model = NimbusBucketRange
+        fields = (
+            "randomizationUnit",
+            "namespace",
+            "start",
+            "count",
+            "total",
+        )
+
+
+class NimbusBranchSerializer(serializers.ModelSerializer):
+    feature = serializers.SerializerMethodField()
+
+    class Meta:
+        model = NimbusBranch
+        fields = ("slug", "ratio", "feature")
+
+    def get_feature(self, obj):
+        return {
+            "featureId": obj.experiment.feature_config.slug,
```

Comment:
We're linking to the feature Id based on its slug, and not its id?

Reply:
Yep in the client all identifiers are slugs

Reply:
And in the data science tooling as well

Reply:
Okey, so this is immutable?

Reply:
We'll have to find a way to enforce that in the workflow since it'll have to match against expected identifiers in the client.  @k88hudson and I have talked about clients publishing a list of known identifiers, but I think we'll have to work that out with the client team later.